airbyte-webapp/nginx/default.conf.template

```diff
@@ -29,4 +33,8 @@ server {
     location = /50x.html {
         root   /usr/share/nginx/html;
     }
+
+    location /api/ {
```

Comment:
Going to write out my understanding so that you can correct me if I'm misinterpreting...

* The UI is already (by default) hosted on port 8000. We want to make it so that the API is also accessible on port 8000 (instead of 80001).
* We do this by adding a rule in the nginx server that runs on port 8000 saying whenever you traffic to `/api` proxy that to the server. so now localhost:8000/api/foo gets forwarded to the server. right so far?
* What I'm not fully understanding is this part of your explanation: "To "enable" this, all you need is to set API_URL=/api/v1/ in your .env file." It seems like as configured it already achieves what you want it to. Why would one need to set it `/api/v1/` in addition?

Reply:
If you don't set `API_URL` it will query localhost:8001 still from the UI instead of querying the now forwarded address.

Reply:
We don't need to make this the default for now, but I could see it being the default in the future.

For SM all I'll need to do is update the .env after upgrading them to a release on a version with this PR.